core/templates/dev/head/pages/community-dashboard-page/services/question-suggestion.service.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview A service for handling question suggestions.
+ */
+
+angular.module('oppia').factory('QuestionSuggestionService', [
+  '$http', function($http) {
+    return {
+      addSuggestion: function(question, associatedSkill, topicName) {
```

Comment:
I think this sounds better? 
```suggestion
      submitQuestionSuggestion: function(question, associatedSkill, topicName) {
```

Reply:
Sure I'll update the verb to "submit". "Question" seems redundant here though since the method will be invoked with the outer class name, i.e. 'QuestionSuggestionService.submitQuestionSuggestion'

Reply:
Ah sounds good.